src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx

```diff
@@ -104,28 +104,41 @@ There are no screenshots, icons, details, or what's new features. To add these,
+| _additionalInfo.md_ | An optional markdown file for any additional information about using your application                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| _screenshots_       | A directory that contains screenshots of your Nerdlets or visualizations. This can contain no more than 6 images. All screenshots must meet the following criteria:<br /><br /><ul><li>3:2 aspect ratio</li><li>PNG format</li><li>landscape orientation</li><li>1600 to 2400 pixels wide</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
I should mention here that the is only in the diff because my editor decided to auto-format this table. I did not write any of this.

That being said, I do like your suggestion. @alexronquillo sorry to tag you so much 😜 but could you weigh in here?